dev/flipbook.tsx

```diff
@@ -242,15 +241,10 @@ function SideBySideQuestionRenderer({
                     background: "#f8f8f8",
                 }}
             >
-                <GradableRenderer
-                    question={question}
-                    apiOptions={{...apiOptions, flags: {mafs: false}}}
-                />
```

Comment:
This is the "no mafs" view when comparing the before and after in flipbook. Removing the "before" view here.